Code/GraphMol/DistGeomHelpers/Embedder.h

```diff
@@ -140,7 +141,8 @@ struct RDKIT_DISTGEOMHELPERS_EXPORT EmbedParameters {
       unsigned int ETversion = 1,
       const DistGeom::BoundsMatrix *boundsMat = nullptr,
       bool embedFragmentsSeparately = true, bool useSmallRingTorsions = false,
-      bool useMacrocycleTorsions = false, bool useMacrocycle14config = false,
+      bool useMacrocycleTorsions = false, double boundsMatForceScaling = 1.0,
```

Comment:
you can't introduce new parameters in such a way that they break the current API.
Please add the new parameter at the end of the argument list